emoji/src/EmojiSearchTable/EmojiSearchTable.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+
+import style from './style.module.css';
+import EmojiSearch from '../EmojiSearch';
+import EmojiTable from '../EmojiTable';
+import emojiList from '../emojiList.json';
+
+export default function EmojiSearchTable(props) {
```

Comment:
Try to use [memo](https://ru.reactjs.org/docs/react-api.html#reactmemo) for **all** components which works with props